packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -46,11 +47,12 @@ import {
   ClientSessionInformationEvent,
   LivenessResponseStream,
 } from '@aws-sdk/client-rekognitionstreaming';
-import { STATIC_VIDEO_CONSTRAINTS } from '../../StartLiveness/helpers';
+import { STATIC_VIDEO_CONSTRAINTS } from '../../utils/helpers';
 import { WS_CLOSURE_CODE } from '../utils/constants';
 
-export const MIN_FACE_MATCH_TIME = 500;
+export const MIN_FACE_MATCH_TIME = 1000;
```

Comment:
I will just double confirm that we actually want to enforce user to stay in oval for 1sec v/s just waiting for 1sec before we display colors. (IMO it should be the former but would check with product)